test/integration/test_conferences.py

```diff
@@ -0,0 +1,488 @@
+"""
+Integration tests for Bandwidth's Voice Voice Conferences API
+"""
+
+from test.utils.env_variables import *
+from test.utils.call_cleanup import callCleanup
+import json
+import time
+from typing import List, Tuple
+import unittest
+from hamcrest import assert_that, has_properties, not_none, instance_of, greater_than
+
+
+import bandwidth
+from bandwidth.api import calls_api
+from bandwidth.model.create_call import CreateCall
+from bandwidth.model.create_call_response import CreateCallResponse
+from bandwidth.model.call_state import CallState
+from bandwidth.model.call_state_enum import CallStateEnum
+from bandwidth.model.update_call import UpdateCall
+from bandwidth.model.redirect_method_enum import RedirectMethodEnum
+from bandwidth.api import conferences_api
+from bandwidth.model.conference_state_enum import ConferenceStateEnum
+from bandwidth.model.conference_recording_metadata import ConferenceRecordingMetadata
+from bandwidth.model.update_conference import UpdateConference
+from bandwidth.model.update_conference_member import UpdateConferenceMember
+from bandwidth.model.file_format_enum import FileFormatEnum
+from bandwidth.rest import RESTClientObject, RESTResponse
+from bandwidth.exceptions import ApiException, UnauthorizedException, ForbiddenException, NotFoundException
+
+class ConferencesIntegration(unittest.TestCase):
+    """
+    Voice Conferences API integration test
+    """
+
+    def setUp(self):
+        """
+        Set up for our tests by creating the CallsApi and ConferencesApi instances
+        for testing as well as the unauthorized and forbidden credentials for the 4xx tests.
+        """
+        configuration = bandwidth.Configuration(
+            username = BW_USERNAME,
+            password = BW_PASSWORD
+        )
+        api_client = bandwidth.ApiClient(configuration)
+
+        # Two Valid API Clients
+        self.calls_api_instance = calls_api.CallsApi(api_client)
+        self.conference_api_instance = conferences_api.ConferencesApi(api_client)
+
+        # Unauthorized API Client
+
+        unauthorizedConfiguration = bandwidth.Configuration(
```

Comment:
Inconsistent `#` comment spacing - IMO your variable names are specific enough that we dont need these comments

Reply:
Good catch, I removed most of those comments.